src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BAll_NUMBER = 101;

```

Comment:
Why it is `public`?

Reply:
Also it is quite strange to store a variable with `101` value.
I think, it would be better to store 100 value and add 1 when you call `nextInt`,
```
MAX_BAll_NUMBER = 100;
...
random.nextInt(MAX_BAll_NUMBER + 1);
```